windows/wix.json

```
@@ -32,6 +32,19 @@
   "directories": [
     {
       "name": "plugins"
+    },
+    {
+      "name": "storage"
+    }
+  ],
+  "environments": [
+    {
+      "name": "OIQ_OTEL_COLLECTOR_STORAGE",
+      "value": "C:\\Program Files\\observIQ OpenTelemetry Collector",
+      "permanent": "yes",
+      "system": "yes",
+      "action": "set",
+      "part": "last"
```

```suggestion
      "part": "all"
```
My understanding is that putting this as "all" will replace the environment variable on install, which is what we want (we don't want to append to the current value of the variable)

https://wixtoolset.org/documentation/manual/v3/xsd/wix/environment.html